operations/reservations.md

```diff
@@ -583,6 +584,7 @@ Adds the specified reservations as a single group. If `GroupId` is specified, ad
 | `TimeUnitAmount` | [Amount](services.md#amount-parameters) | optional | Amount of each night of the reservation. |
 | `TimeUnitPrices` | array of [Time unit amount parameters](reservations.md#time-unit-amount-parameters) | optional | Prices for time units of the reservation. E.g. prices for the first or second night. |
 | `ProductOrders` | array of [Product order parameters](services.md#product-order-parameters) | optional | Parameters of the products ordered together with the reservation. |
+| `CreditCardId` | string | optional | Identifier of [Credit card](finance.md#credit-card) belonging to [Customer](customers.md#customer) who owns the reservation. |
```

Comment:
CC @vitasamek  - I'd add something about the fact, that this credit card is for settlement of the reservation. E.g. `Identifier of Credit card (belonging to reservation owner) that may be used for settlement of the reservation.

Reply:
Maybe I'd also move it below CompanyId, not leave it on the bottom...